Content.Server/Singularity/Components/ServerSingularityComponent.cs

```diff
@@ -103,7 +103,7 @@ public void Update(int seconds)
         protected override void OnRemove()
         {
             _playingSound?.Stop();
-            SoundSystem.Play(Filter.Pvs(Owner), _singularityCollapsingSound.GetSound(), Owner.Transform.Coordinates);
+            SoundSystem.Play(Filter.Pvs(Owner), _singularityCollapsingSound.GetSound(), Owner);
```

Comment:
In this case the singulo is probably being deleted so probably not suitable to use the entity.